recipes/vpselector/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "vpselector" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/vpselector-{{ version }}.tar.gz
+  sha256: 18f9e3408cd33309e9c486d26986b7bc049cffed99312e364422a60fc5671cad
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pandas >=1.1.5
+    - pyqt >=5.15.0
+    - matplotlib-base >=3.7.1
```

Maybe this fixes the linux failure.

```suggestion
    - matplotlib >=3.7.1
```